rfcs/formats/identity.md

```diff
@@ -0,0 +1,203 @@
+<!--
+ Copyright IBM Corp. All Rights Reserved.
+
+ SPDX-License-Identifier: CC-BY-4.0
+ -->
+# Identity Plane Data Formats
+
+* Authors: Bishakh Chandra Ghosh, Venkatraman Ramakrishna, Krishnasuri Narayanam
+* Status: Draft
+* Since: 25-July-2021
+
+
+## Participant Unit Identity
+
+Each network participant (individual/organization) has its identity on the identity plane as *Participant Unit Identity*, in the form of a **DID**. We also refer to it as *Participant Unit DID*.
+The participant DID Document must specify one verification method with [authentication](https://w3c.github.io/did-core/#authentication) verification relationship. The *Participant Unit DID* must be registered in some IIN registry.
+
+### Example:
+
+```json
+{
+  "id": "did:exampleiin:org1",
+  "verificationMethod": [{
+    "id": "did:exampleiin:org1#key1",
+    "type": "Ed25519VerificationKey2020",
+    "controller": "did:exampleiin:org1",
+    "publicKeyMultibase": "zH3C2AVvLMv6gmMNam3uVAjZpfkcJCwDwnZn6z3wXmqPV"
+  }],
+
+  "authentication": ["did:exampleiin:org1#key1"]
+}
+```
+
+
+## Network Identity
+
+The identity of a network as a single entity is represented by the *Network Identity* which is a DID called *Network DID*.
+
+A permissioned blockchain network is formed by mutual agreement among its participants. The network as an entity is thus not controlled by any single member but together by its participant units, which may be individuals or companies or organizations. But inspite of having multiple controllers, the network as an entity must have its own identity, through which it can be discovered. In this specification, each permissioned blockchain network which needs to exchange its identity for interoperation has a Network DID and a corresponding Network DID Document, that we describe in this section.
+
+A permissioned blockchain network has a Network DID that represents it.
+The Network DID document conforms to the [did-core](https://www.w3.org/TR/did-core) specifications.
+
+**id** - This property is mandatory which is the DID URI for the network. This id (DID) can be used to resolve the DID Document of the network from the IIN DID registry.
+
+**verificationMethod** - One verification method property must be present in the Network DID document and it must be able to support [Group Control](https://www.w3.org/TR/did-core/#group-control), so that the network participant units can jointly control the Network DID Document. Additionally, the verification method must specify who can update the DID in case the network changes as participant units leave and new ones join. We describe a multi-signature based verificationMethod type in the following sections.
```

Comment:
Apart from "jointly control", should we add "jointly validate the network's composition"?

Reply:
"verificationMethod" should be in code font, right?